projects/packages/forms/changelog/add-project-for-jetpack-inspect

```diff
@@ -0,0 +1,4 @@
+Significance: minor
+Type: added
+
+Added a new project for Jetpack Inspect.
```

Comment:
Change entry should reflect the effect on _this_ project.
```suggestion
Significance: patch
Type: added
Comment: Prettier reformatted some code. No change to the code itself.

```
Of course, doing that will then take the version back down to 0.19.8-alpha, so all the bumps below will get undone too.